README.md

```diff
@@ -23,19 +23,6 @@ boards.
 
 There are multiple types of layers included here.
 
-SoC-oriented:
-
-- meta-freescale
-- meta-sunxi
-- meta-rockchip
-- meta-renesas
-
-Vendor-oriented:
-
-- meta-variscite
-- meta-phytec
-- meta-toradex-nxp
-
```

Comment:
Definitely want to keep everything around and give people a chance to update. So drop this commit